TestingTravis/TestClass.swift

```diff
@@ -11,9 +11,11 @@ import Foundation
 class TestClass: NSObject {
     
     var forceUnwrapMe: String?
+    var forceUnwrap2: Int?
     
     override init() {
         print(forceUnwrapMe!)
+        print(forceUnwrap2!)
     }
     
```

Comment:
<table data-meta="generated_by_danger">
  <tbody>
    <tr>
      <td>:warning:</td>
      <td width="100%" data-sticky="false"><span data-href="https://github.com/Nakul97/TestingTravis/blob/a9f13a523c5d48fe381b2f932ba2d4c6634e633f/TestingTravis/TestClass.swift#L20"></span>Lines should not have trailing whitespace.<br />
<code>trailing_whitespace</code> <code>TestClass.swift:20</code></td>
    </tr>
  </tbody>
</table>